src/ui/ui_exports/ui_export_types/style_sheet_paths.test.js

```diff
@@ -21,51 +21,77 @@ import { resolve } from 'path';
-      .toThrowError(/\[plugin:test\] uiExports.styleSheetPaths must be an absolute path/);
+    expect(() => styleSheetPaths([], 'public/bar.css', 'styleSheetPaths', pluginSpec)).toThrowError(
+      /\[plugin:test\] uiExports.styleSheetPaths must be an absolute path/
+    );
   });
 
   it('path must be child of public path', () => {
-    expect(() => styleSheetPaths([], '/another/public/bar.css', 'styleSheetPaths', pluginSpec))
-      .toThrowError(/\[plugin:test\] uiExports.styleSheetPaths must be child of publicDir/);
+    expect(() =>
+      styleSheetPaths([], '/another/public/bar.css', 'styleSheetPaths', pluginSpec)
+    ).toThrowError(/\[plugin:test\] uiExports.styleSheetPaths must be child of publicDir/);
   });
 
   it('only supports css or scss extensions', () => {
-    expect(() => styleSheetPaths([], '/kibana/public/bar.bad', 'styleSheetPaths', pluginSpec))
-      .toThrowError('[plugin:test] uiExports.styleSheetPaths supported extensions [.css, .scss], got ".bad"');
+    expect(() =>
+      styleSheetPaths([], '/kibana/public/bar.bad', 'styleSheetPaths', pluginSpec)
+    ).toThrowError(
+      '[plugin:test] uiExports.styleSheetPaths supported extensions [.css, .scss], got ".bad"'
+    );
   });
 
   it('provides publicPath for scss extensions', () => {
     const localPath = resolve(dir, 'kibana/public/bar.scss');
     const uiExports = styleSheetPaths([], localPath, 'styleSheetPaths', pluginSpec);
 
-    expect(uiExports.styleSheetPaths).toHaveLength(1);
-    expect(uiExports.styleSheetPaths[0].localPath).toEqual(localPath);
-    expect(uiExports.styleSheetPaths[0].publicPath).toEqual('plugins/test/bar.css');
+    expect(uiExports.styleSheetPaths).toMatchInlineSnapshot(`
+Array [
+  Object {
+    "localPath": <absolute>/kibana/public/bar.scss,
+    "publicPath": "plugins/test/bar.css",
+  },
+]
+`);
   });
 
   it('provides publicPath for css extensions', () => {
     const localPath = resolve(dir, 'kibana/public/bar.scss');
     const uiExports = styleSheetPaths([], localPath, 'styleSheetPaths', pluginSpec);
 
-    expect(uiExports.styleSheetPaths).toHaveLength(1);
-    expect(uiExports.styleSheetPaths[0].localPath).toEqual(localPath);
-    expect(uiExports.styleSheetPaths[0].publicPath).toEqual('plugins/test/bar.css');
+    expect(uiExports.styleSheetPaths).toMatchInlineSnapshot(`
+Array [
+  Object {
+    "localPath": <absolute>/kibana/public/bar.scss,
+    "publicPath": "plugins/test/bar.css",
+  },
+]
+`);
   });
 
   it('should normalize mixed slashes', () => {
     const localPath = resolve(dir, 'kibana/public\\bar.scss');
     const uiExports = styleSheetPaths([], localPath, 'styleSheetPaths', pluginSpec);
 
-    expect(uiExports.styleSheetPaths).toHaveLength(1);
-    expect(uiExports.styleSheetPaths[0].localPath).toEqual(localPath);
+    expect(uiExports.styleSheetPaths).toMatchInlineSnapshot(`
+Array [
+  Object {
+    "localPath": <absolute>/kibana/public\\bar.scss,
```

Comment:
@tylersmalley Does this test look right to you? I'm pretty surprised to see that `resolve()` is not converting `\\` to `/`, and I'm pretty sure these results are contrary to the description of the test, do you think this is a problem?

Reply:
No issue, just wanted to make sure things were working fine, if you think it's alright then I can just merge this PR and my subsequent PR won't introduce this somewhat strange snapshot.

Reply:
Ok, then I agree it should block this PR since it's not changing behavior.